aptos-move/move-examples/mint_nft_v2/1-Create-NFT/sources/create_nft.move

```diff
@@ -0,0 +1,187 @@
+///         aptos account lookup-address --profile nft-receiver
+///         {
+///           "Result": "e9907369a82cc0d5b93c77e867e36b7e412912ed4825e17b3ca49541888cae67"
+///         }
+///
+/// 3.b Run the `delayed_mint_event_ticket` function below. Note this is referred to as delayed because it requires
+///     the asynchronous (more specifically, an off-chain) approval of the module_owner in order to mint.
+///
+///     Run the following command:
+///         `aptos move run --function-id default::create_nft::delayed_mint_event_ticket --args address:nft-receiver --profile default`
+///
+///     Sample output is below:
+///         aptos move run --function-id default::create_nft::delayed_mint_event_ticket --args address:nft-receiver --profile default
+///
+///         Do you want to submit a transaction for a range of [52300 - 78400] Octas at a gas unit price of 100 Octas? [yes/no] >
+///         yes
+///         {
+///           "Result": {
+///             "transaction_hash": "0xd3f7bab04e18f4e631f40e669bf1e3224d8c08318da509aeedf1806cc0cd6cdd",
+///             "gas_used": 523,
+///             "gas_unit_price": 100,
+///             "sender": "c9f5ffb4fa164729e28733d5878ee4e90604478640dfb5ccb477452c0fe1cd79",
+///             "sequence_number": 18,
+///             "success": true,
+///             "timestamp_us": 1683077797249592,
+///             "version": 3652742,
+///             "vm_status": "Executed successfully"
+///           }
+///         }
+///
+///     View the transaction on the explorer here:
+///     https://explorer.aptoslabs.com/txn/0xd3f7bab04e18f4e631f40e669bf1e3224d8c08318da509aeedf1806cc0cd6cdd?network=devnet
+module mint_nft_v2::create_nft {
+    use std::bcs;
+    use std::error;
+    use std::signer;
+    use std::object;
+    use std::string::{Self, String};
+    use aptos_framework::account;
+
+    use aptos_token_objects::aptos_token::{Self, AptosToken};
+    use aptos_token_objects::collection;
+
+    // This struct stores an NFT collection's relevant information
+    struct ModuleData has key {
+        collection_name: String,
+        creator: address,
+        token_description: String,
+        token_name: String,
+        token_uri: String,
+        property_keys: vector<String>,
+        property_types: vector<String>,
+        property_values: vector<vector<u8>>,
+        collection_object_address: address,
+    }
+    /// Action not authorized because the signer is not the admin of this module
+    const ENOT_AUTHORIZED: u64 = 1;
+
+    /// `init_module` is automatically called when publishing the module.
+    /// In this function, we create an example NFT collection and an example token.
+    fun init_module(source_account: &signer) {
+        let collection_name = string::utf8(b"Collection name");
+        let description = string::utf8(b"Description");
+        let collection_uri = string::utf8(b"Collection uri");
+        let token_name = string::utf8(b"Token name");
```

Comment:
Why not just make these constants? These are hardcoded anyway

Reply:
yea I'll add that and explain how you could pass in those arguments if you want but we'll use constants for simplicity